relation-extraction/src/main/java/org/cogcomp/re/RelationAnnotator.java

```diff
@@ -188,7 +169,7 @@ public void addView(TextAnnotation record) throws AnnotatorException {
                 }
             }
         }
-        record.addView(ViewNames.RELATION + "_" + type, mentionView);
+        record.addView(ViewNames.RELATION, mentionView);
```

Comment:
here `mentionView` was initialized with `ViewNames.MENTION`, and now you are adding it to `ViewNames.RELATION`

Reply:
Ah I see. I think it won't make any big problems. Will remember it ...

Reply:
Ok